module/platform/darwin/SensorFilter/src/SensorFilter.cpp

```diff
@@ -267,15 +266,15 @@ namespace module::platform::darwin {
                     NUClear::log<NUClear::WARN>(makeErrorString("Right FSR", input.fsr.right.error_flags));
                 }
 
-                // Read through all of our sensors
-                for (uint32_t i = 0; i < 20; ++i) {
-                    auto& original = utility::platform::darwin::getDarwinServo(i, input);
+                // Loop through all the servos, emitting a message::input::Sensors for each one
+                for (uint32_t id = 0; id < 20; ++id) {
+                    auto& original = utility::platform::darwin::getDarwinServo(id, input);
                     auto& error    = original.error_flags;
 
                     // Check for an error on the servo and report it
                     while (error != DarwinSensors::Error::OK) {
```

Comment:
Could this be an `if`, or am I missing something?

Reply:
Yeah, I have no idea why this is a while, it unconditionally breaks at the end of the first loop...

Reply:
https://github.com/NUbots/NUbots/commit/2c98722097ac19311860f2e90ea88617fabeb1d4
Apparently @JakeFountain did it :P